registration/settings.py

```diff
@@ -113,8 +126,35 @@
 
 USE_TZ = True
 
+LANGUAGES = [
+    ('de', _('German')),
+    ('en', _('English')),
```

Comment:
Do we really want to translate it? (is it worth the effort?)

Reply:
Hm, I always do that because German+English mixup throughout the code is just so ugly :D Actually it's not that much work translating the models and stuff. So I would keep it this way.

Reply:
ok